src/TLuaInterpreter.cpp

```diff
@@ -17295,7 +17365,7 @@ void TLuaInterpreter::loadGlobal()
         qt_ntfs_permission_lookup++;
 #endif
         if (!(QFileInfo(pathFileName).isReadable())) {
-            failedMessages << tr("%1 (isn't a readable file or symlink to a readable file)", "This is not a file or a symbolic link to a file").arg(pathFileName);
+            failedMessages << tr("%1 (isn't a readable file or symlink to a readable file)", "This is not a file or a symbolic link to a readable (for this user) file").arg(pathFileName);
```

Comment:
I feel like this is more specificity than is needed and is more likely to confuse the issue than help most people.

Reply:
It is only going to show if there is a problem - and if there is a problem it helps to know as much detail as we can reasonable establish - in this case that there is something there but it isn't a file or symlink to a file we can read...

Reply:
I don't think so. To be honest "This is not a file or a symbolic link to a readable (for this user) file" makes little sense when you read it.

Reply:
Ha, it is a comment for the translators not a UI text. I suppose I could cut it out entirely and let them work it out from the Engineering text...

Reply:
It basically repeats the first text, not adding any more information

Reply:
Did you mean: "the explanation text for the translators does not add any more information compared to the source text"? Well it does explicitly point out that the problem is likely to be permissions (*read-permissions* for *this* user") - I do not have the source open around this - but - IIRC - there is a prior test that checks for the *existence* of the file prior to this check that the user can read it...